phnt/include/ntrtl.h

```diff
@@ -9555,10 +9557,11 @@ RtlDecodePointer(
  * @sa https://learn.microsoft.com/en-us/previous-versions/bb432255(v=vs.85)
  */
 NTSYSAPI
+_Ret_maybenull_
```

Comment:
Need to be the first line above NTSYSAPI

Reply:
Committed.